README.md

```diff
@@ -72,13 +72,18 @@ plugins: [
       // Optional: Specify a number to perform http-retries for network failures. By default it is set to 3.
       httpRetries: 2,
 
-      //Specify the number of entries/assets to be fetched per page. Maximum limit accepted is 100. By default it is set to 50.
+      //Optional: Specify the number of entries/assets to be fetched per page. Maximum limit accepted is 100. By default it is set to 50.
       limit: 50
+
+      //Optional: Specify list of headers to be passed to Contentstack API.
+      enableEarlyAccess : ['newcda','taxonomy']
```

Comment:
We need to take two different env keys.
`enableEarlyAccessKey`
`enableEarlyAccessValue`